de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/impl/PaySelectionUpdater.java

```diff
@@ -153,6 +175,7 @@ public void run(final String localTrxName) throws Exception
 				setTrxName(localTrxName);
 
 				update0();
+				CacheMgt.get().reset();
```

Comment:
why full cache reset is needed?

Reply:
i would really really really avoid this.

Reply:
forgot that one there while testing smth.